core/client/db/Cargo.toml

```diff
@@ -14,7 +14,7 @@ kvdb-memorydb = { git = "https://github.com/paritytech/parity-common", rev="b031
 linked-hash-map = "0.5"
 hash-db = { version = "0.14.0" }
 primitives = { package = "substrate-primitives", path = "../../primitives" }
-runtime_primitives = { package = "sr-primitives", path = "../../sr-primitives" }
+sr_primitives = { package = "sr-primitives", path = "../../sr-primitives" }
```

Comment:
Package not needed here.

Reply:
oh weird, I grep-ed the whole package for such, some must have been changed afterward.